### PR TITLE
Docs alerting: meta monitoring topics

### PR DESCRIPTION
**Pull request by brendamuir**
Adds insights logs
Updates metamonitoring and metrics/notes
Moves meta monitoring to a separate monitor section

### CODE REVIEW

File: docs/sources/alerting/monitor/_index.md
Comment (jdbaldry) [inline docs/sources/alerting/monitor/_index.md:26]: Relref links shouldn't have trailing slashes (https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash).
```suggestion
[meta-monitoring]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/monitor/meta-monitoring"
[meta-monitoring]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/meta-monitoring"
```

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (jdbaldry) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:4]: This currently redirects https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ to https://grafana.com/docs/grafana/latest/alerting/set-up/meta-monitoring/.

Keeping the same alias will mean that  https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ will now redirect to your new page location https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/.

If the original redirect was correct and desirable, then I believe this alias is still correct but I wanted to double check that the original redirect is correct.

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (jdbaldry) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:3]: I'm not sure this alias is doing anything. I am pretty sure it would have been redirecting https://grafana.com/docs/grafana/latest/alerting/set-up/meta-monitoring/ to itself.

I think it can be removed unless it was actually meant to redirect something like https://grafana.com/docs/grafana/latest/alerting/meta-monitoring/ to this page?

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (jdbaldry) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:5]: I believe the intent of this alias is to redirect the old location of https://grafana.com/docs/grafana/latest/alerting/set-up/meta-monitoring/ to https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/.

The alias to do that is:

```suggestion
  - ../set-up/meta-monitoring/
```

**Review by jdbaldry — COMMENTED** [review]
I've just reviewed the front matter and `docs/reference` shortcode.

Let me know if you want me to take a look at the rest of the content.

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (brendamuir) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:4]: this is what i want to happen: that this page https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ redirects to your new page location https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/.

-> what do i need to change?

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (brendamuir) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md]: done!

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (09jvilla) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:86]:
```suggestion
## Metrics for Mimir-managed alerts
```

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (09jvilla) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:92]:
```suggestion
To meta monitor Grafana Mimir-managed alerts, open source and on-premise users need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by the Mimir ruler.
```

Comment (09jvilla) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:92]: Add something like 

Grafana Cloud customers can find this metric in their `grafanacloud-usage` datasource.

Comment (09jvilla) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:92]: note that its going to have a slightly different name though so that might cause confusion. In Grafana Cloud its:
`grafanacloud_instance_rule_evaluation_failures_total:rate5m{}`

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (09jvilla) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:106]:
```suggestion
To meta monitor the Alertmanager, you need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by Alertmanager.
```

Comment (09jvilla) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:106]: note that this is only true for oss or on-prem Enterprise customers. If you're cloud, we provide that database for you. :)

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (09jvilla) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:212]: would it make sense to put this in a dedicated code block?

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (09jvilla) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:216]: Do we want a list with just 1 number in it? Or should this be step 2 and "You can see" be step 3?

**Review by 09jvilla — CHANGES_REQUESTED** [review]
I focused on reviewing the 'usage insights' content and overall looks good minus a couple small suggestions. 

I'm honestly a bit confused about what content for metamonitoring will live in the OSS docs vs the Cloud docs, especially since some of it (like the usage insights bit) is only relevant for Cloud customers. This seems to me like a case where you wouldn't want any mention of it in the OSS Grafana docs. But will defer to you as to what content should go where.

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (09jvilla) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:218]: Do you think it would be helpful to have a screenshot here?
If so, here's a link to one: https://drive.google.com/file/d/1wuINfWpQk175SCnQrlAcUAiGlR42pYXa/view?usp=sharing

Comment (09jvilla) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:218]:
```suggestion
   You can see the rule contents (in the `rule` field), the rule name (in the `name` field), the name of the group it’s in (in the `group` field), and the error message (in the `err` field).
```

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (brendamuir) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md]: done

**Comment by brendamuir** [comment]
Hey Jen-yes I'll fix that as a next step. I have other alerting content to split out into cloud only and i figured i'd do it at once. for now-i've made sure it's clearly marked with notes-hope that's clear enough for now?

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (grobinson-grafana) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:34]:
```suggestion
There are no metrics in the `grafanacloud-usage` data source at this time that can be used to meta monitor Grafana-managed alerts.
```

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (grobinson-grafana) [inline docs/sources/alerting/monitor/meta-monitoring/_index.md:89]:
```suggestion
These metrics are available in OSS, on-premise, and Grafana Cloud.
```

Comment (grobinson-grafana) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:89]: There is more than one metric in total, although for some reason I documented just one of them when I wrote this page. Not sure if we should change it from singular to plural in that case?

File: docs/sources/alerting/monitor/_index.md
Comment (grobinson-grafana) [inline docs/sources/alerting/monitor/_index.md:14]: Do you also want to mention insights logs?

**Review by grobinson-grafana — APPROVED** [review]

File: docs/sources/alerting/monitor/_index.md
Comment (brendamuir) [reply docs/sources/alerting/monitor/_index.md:14]: i think i'll leave it out now - also have left out alerting insights until that's merged - and until i split it into cloud only docs

File: docs/sources/alerting/monitor/meta-monitoring/_index.md
Comment (jdbaldry) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:3]: Cool, I believe that this alias is still correct and will redirect https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ to https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/ once this PR is merged and backported.

Comment (jdbaldry) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:3]: You can check locally by running `make docs` and browsing to http://localhost:3002/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/. You should be redirected to your new page. If it 404s, the alias is incorrect.

Comment (brendamuir) [reply docs/sources/alerting/monitor/meta-monitoring/_index.md:3]: cool, thanks

**Review by jdbaldry — APPROVED** [review]

**Comment by 09jvilla** [comment]
> Hey Jen-yes I'll fix that as a next step. I have other alerting content to split out into cloud only and i figured i'd do it at once. for now-i've made sure it's clearly marked with notes-hope that's clear enough for now?

Works for me!

**Review by 09jvilla — APPROVED** [review]
LGTM

**Pull request merged**
